### ISSUE TITLE
Accessibility issue: Search input doesn't have label

### ISSUE THREAD

**Issue by zhr1130** [react-bootstrap-table/react-bootstrap-table2#930]
We got accessibility warning for the search input:

Rule
label: Form elements must have labels  WCAG 1.3.1, WCAG 3.3.2

How to fix 
Fix ONE of the following:
aria-label attribute does not exist or is empty
aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
Form element does not have an implicit (wrapped) <label>
Form element does not have an explicit <label>
Element has no title attribute or the title attribute is empty

Snippet 
<input type="text" class="form-control detail-search" placeholder="Search" value="">
Path | .card:nth-child(2) > .card-body > div > .form-control.detail-search[placeholder="Search"]

**Comment by zhr1130** [comment]
I created a PR to fix it: https://github.com/react-bootstrap-table/react-bootstrap-table2/pull/931
Can someone take a look ?

**Comment by AllenFang** [comment]
Fixed it and please upgrade below packages to newest:
https://react-bootstrap-table.github.io/react-bootstrap-table2/blog/2019/05/19/version-bump.html
